docroot/modules/custom/civictheme_content/modules/civictheme_content_default/civictheme_content_default.post_update.php

```diff
@@ -24,6 +26,8 @@ function civictheme_content_default_post_update_set_homepage() {
 
 /**
  * Provisions links in footer and updates menu block configurations.
+ *
+ * @SuppressWarnings(PHPMD.StaticAccess)
  */
 function civictheme_content_default_post_update_provision_footer_links() {
   $theme = \Drupal::theme()->getActiveTheme()->getName();
```

Comment:
@rebelmusic 
Could you please expand this static access to use `\Drupal::\Drupal::getContainer()->get('theme.manager')`

Reply:
i chose to use Drupal::service('theme.manager'). Let me know if that is ok